loadtest/control/simplecontroller/controller.go

```diff
@@ -179,9 +179,11 @@ func (c *SimpleController) createActions(definitions []actionDefinition) error {
 			return fmt.Errorf("could not find action %q", def.ActionId)
 		}
 
-		if def.RunPeriod < 1 {
-			mlog.Warn(fmt.Sprintf("could not create action from %q, run period needs to be >= 1", def.ActionId))
+		if def.RunPeriod == 0 {
+			mlog.Warn(fmt.Sprintf("skipping action %s, run period is: %d", def.ActionId, def.RunPeriod))
```

Comment:
Again, this is my fault for proposing it. 
I think we agreed to actually remove the warning since we accept 0 to be valid in a way.